### ISSUE TITLE
Insiders version isn't loaded correctly after update

### ISSUE THREAD

**Issue by Ungerfall** [microsoft/azuredatastudio#7562]
- Azure Data Studio Version: Unable to see version due to button inaccessibility, but /bin/azuredatastudio-insiders says COMMIT="e4e71af597d027c3a46ae33276e94d4681429619"

Screenshot:
![image](https://user-images.githubusercontent.com/8143895/66389525-ac55f400-e9d9-11e9-9f8c-d7e84a1669e8.png)

upd: reinstallation does not help

**Comment by jtsom** [comment]
Same happening under Windows - completely blank screen. Unusable.

$ azuredatastudio-insiders --version
1.12.0-insider
e4e71af597d027c3a46ae33276e94d4681429619
x64

**Comment by jtsom** [comment]
More info from --verbose:
```
$ azuredatastudio-insiders --verbose

[main 2019-10-08T13:30:22.498Z] Starting VS Code
[main 2019-10-08T13:30:22.498Z] from: c:\Users\John Tsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app
[main 2019-10-08T13:30:22.498Z] args: { _: [], verbose: true, diff: undefined }
[main 2019-10-08T13:30:22.500Z] Resolving machine identifier...
[main 2019-10-08T13:30:22.500Z] Resolved machine identifier: fa0b240262cb9e8245f9049f5e940b8540bd0f2964c210d59884b97c556370a7 (trueMachineId: undefined)
[main 2019-10-08T13:30:22.518Z] update#setState idle
[main 2019-10-08T13:30:22.519Z] [storage state.vscdb] open(c:\Users\John Tsombakos\AppData\Roaming\azuredatastudio\User\globalStorage\state.vscdb, retryOnBusy: true)
[main 2019-10-08T13:30:22.520Z] lifecycle (main): phase changed (value: 2)
[main 2019-10-08T13:30:22.521Z] windowsManager#open
[main 2019-10-08T13:30:22.617Z] lifecycle (main): phase changed (value: 3)
[main 2019-10-08T13:30:22.621Z] telemetry/optInStatus { optIn: true }
[main 2019-10-08T13:30:22.621Z] telemetry/update:win32SetupTarget { target: 'user' }
[main 2019-10-08T13:30:22.622Z] telemetry/machineIdFallback { usingFallbackGuid: false }
[4388:1008/093022.716:ERROR:gpu_process_transport_factory.cc(1017)] Lost UI shared context.
[main 2019-10-08T13:30:22.761Z] [storage state.vscdb] Trace (event): SELECT * FROM ItemTable
[main 2019-10-08T13:30:22.761Z] [storage state.vscdb] getItems(): 41 rows
[main 2019-10-08T13:30:22.866Z] [storage state.vscdb] updateItems(): insert(Map(2) {telemetry.lastSessionDate => Tue, 08 Oct 2019 13:30:09 GMT, telemetry.currentSessionDate => Tue, 08 Oct 2019 13:30:22 GMT}), delete(Set(0) {})
[main 2019-10-08T13:30:22.868Z] [storage state.vscdb] Trace (event): BEGIN TRANSACTION
[main 2019-10-08T13:30:22.872Z] [storage state.vscdb] Trace (event): INSERT INTO ItemTable VALUES ('telemetry.lastSessionDate','Tue, 08 Oct 2019 13:30:09 GMT'),('telemetry.currentSessionDate','Tue, 08 Oct 2019 13:30:22 GMT')
[main 2019-10-08T13:30:22.875Z] [storage state.vscdb] Trace (event): END TRANSACTION
[main 2019-10-08T13:30:22.893Z] getShellEnvironment: running on Windows, skipping
[4388:1008/093024.079:INFO:CONSOLE(231)] "[uncaught exception]: reflect-metadata shim is required when using class decorators", source: C:\Users\John Tsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\out\bootstrap-window.js (231)
```

**Comment by Charles-Gagnon** [comment]
Publishing new version now - will update when that's out. Apologies for the inconvenience everyone!

**Comment by jtsom** [comment]
Don't forget - you'll have to use the installer - you won't be able to use the in-application updater!

**Comment by jtsom** [comment]
So, not fixed?

**Comment by kburtram** [comment]
@jtsom the insider build with this bug has been unpublished.  Though we still don't have a new insider build out today (the release publishing is stuck in the build queue).  I'm leaving the issue open until a new build is actually available.

**Comment by kburtram** [comment]
Updated insider builds have published, so closing this issue now.

**Comment by Charles-Gagnon** [comment]
Note for anyone hitting this issue - you'll need to uninstall and then reinstall to fix this issue. A couple people have been reporting seeing ADS processes still running after uninstall though (likely because of the bug causing it to not start up) and so we suggest restarting your machine after uninstall just to ensure you have a clean environment when installing the latest insiders version. Let us know if you run into any more issues!
